packages/babel-plugin-transform-typescript/src/index.js

```diff
@@ -266,11 +266,14 @@ export default declare(api => {
       }
     }
 
-    if (binding.identifier.name != "React") {
+    if (
+      binding.identifier.name !== "React" &&
+      binding.identifier.name !== options.jsxPragma
```

Comment:
what about using "React" as default in `options.jsxPragam`, that would simplify the logic a bit.

Reply:
✔️